cmake/modules/AddRootBench.cmake

```diff
@@ -16,19 +16,27 @@ function(RB_ADD_GBENCHMARK benchmark)
   # to implement because some ROOT components create more than one library.
   target_link_libraries(${benchmark} ${ARG_LIBRARIES} gbenchmark RBSupport)
   #ROOT_PATH_TO_STRING(mangled_name ${benchmark} PATH_SEPARATOR_REPLACEMENT "-")
-  #ROOT_ADD_TEST(gbench${mangled_name}
-  #  COMMAND ${benchmark}
-  #  WORKING_DIR ${CMAKE_CURRENT_BINARY_DIR}
-  #  LABELS "benchmark")
+  if(ARG_POSTCMD)
+    set(postcmd POSTCMD ${ARG_POSTCMD})
+  endif()
+  if(flamegraph)
+      set(postcmd ${postcmd} "${PROJECT_SOURCE_DIR}/rootbench-scripts/flamegraph.sh")
```

Comment:
Could you upload the `flamegraph.sh`. Depending on its contents it may deep the postcmd design suboptimal (as it is inserting a sequential point).

Reply:
Can we add it in a second iteration? @HLilit  has it ready..

Reply:
Then the PR would be incomplete, right? If we merge it now it’d be as functional as not merging it, an I missing something?

Reply:
Fine!